### DISCUSSION TITLE
Clicking "missing" next to the "duplicate" job finds that all pictures were "missed" – every time.

### DISCUSSION THREAD

**Discussion by Qhilm** [immich-app/immich#11407] [Q&A]
1. Administration > jobs
2. click "missing" next to "duplicate detection"
3. "Waiting" goes up, reaches <nb of pictures>  according to server stats.

This can be reproduced every time. Is that expected?

**Comment by mertalev** [comment]
No. It might be that the jobs are failing. Do the server logs say anything? Also try clicking "missing" for smart search - it's a prerequisite for duplicate detection.

**Reply by Qhilm** [reply]
the smart search job is running since a while, it's ingestive my google photos takeout at the moment, maybe that's the problem. I'll wait for the smart search job to stop, then will test again.

**Reply by Qhilm** [reply]
Smart Search had an issue, switching model had broken it (even though I had clicked "clear" and ran it again). Now it seems to work normally the postgres container is working its way slowly through all the photos. I will check the results when it's completed.

**Comment by Qhilm** [comment]
ok, it was indeed smart search which was broken, fixing it also fixed the duplicate job – I now am the proud owner of 15% duplicates...
